src/screens/User/AddUserScreen.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import {userModel as model} from "../../models/models";
+import {AddScreen} from "../../components/screens/AddScreen";
+
+const initialProps = {
+    title: "Add " + model.type,
+    fields: model.getFormFields(),
+    redirectPath: model.getBaseUrl()
+};
+
+const mapDispatchToProps = (dispatch) => ({
+    add: (item) => {
+        console.log('add');
```

Comment:
@mtdalton Remove console log

Reply:
This console.log needs to stay here. This is on the UserScreen that I use to test. Seeing as this add function is not hooked up to an actual API, I log 'add' so I know that it works. This code isn't used and is not accessible when using the RAT in projects.